roles/logs/templates/push_log.sh.j2

```diff
@@ -11,7 +11,7 @@ then
             dpkg -l > /var/log/apt/dpkg.list
             pip freeze > /var/log/pip.freeze
             iptables-save > /var/log/iptables.rules
-            rsync -azP /var/log/nginx/* /var/log/kern.log /var/log/messages /var/log/syslog* /var/log/ansible-pull.log /tmp/hdw_info.txt /var/log/ansible-pull.log* /var/log/apt/dpkg.list* /var/log/pip.freeze* /var/log/iptables.rules {{ central_server }}:/ansible/logs/{{ ansible_hostname }}/
+            rsync -e "ssh -o StrictHostKeyChecking=no" -azP /var/log/nginx/* /var/log/kern.log /var/log/messages /var/log/syslog* /var/log/ansible-pull.log /tmp/hdw_info.txt /var/log/ansible-pull.log* /var/log/apt/dpkg.list* /var/log/pip.freeze* /var/log/iptables.rules {{ central_server }}:/ansible/logs/{{ ansible_hostname }}/
```

Comment:
The idea behind this is to avoid 2 things : 

- Using static known_hosts file. As soon as a key changes, rsyncs and sshs will fail. And I thought it'd be cool to forget about that.
- We could create a couple of tasks with ssh-keyscan to fill our known_hosts to avoid this. 1) But, this will not offer more security, as client will accept any new offered key (ssh-keyscan). 2) A couple less network tasks is still good to take.
Now the real issue, yes we will accept any fingerprint. I don't see really any serious issue giving away logs which are mostly meant for our debugging and statistic tasks.
I'll let you guys @fheslouin @letompouce decide on that point, mine is given ;)